content/docs/ops/configuration-management.md

```diff
@@ -47,8 +47,6 @@ is encouraged and qualifies as a review. Review should include assessment of arc
 
 ![Pipeline Example](/img/pipeline-example.png)
 
-A more detailed example of this process can be seen in [Updating Cloud Foundry]({{< relref "updating-cf.md" >}}).
```

Comment:
Ah thanks, good to check. 

Process I use for checking: Generally it's fine to make changes to this doc that are compatible with the requirements in the CM (Configuration Management) family and compatible with our SSP statements in the CM family. Usually this means it's fine to make configuration management _more_ rigorous (or to make changes that have no impact on rigorousness).

In this case, looking at the SSP, CM-8 (1) says "If there are components to update, remove or test, Cloud Operations follows the update procedures listed here: https://cloud.gov/docs/ops/updating-cf/ "

So I'd hesitate to completely dump it, without covering the basics of the procedure or redirecting the page to something that covers the basics.